### PR TITLE
adding my first js code

### PR DESCRIPTION
**Pull request by kumariprity123**

### CODE REVIEW

File: school/sum.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function findSum(x,y){
+    let sum = x+y;
+    return sum;
+}
+var x=6,y=5;
```
Comment (kumariprity123) [inline school/sum.js:5]: please add some space between operator.

Comment (kumariprity123) [inline school/sum.js]: please add space between operator

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
 # codepractice
 This project includes my practice of c++ coding.
+
+steps to clone and open a project into vs code:
+
+1. open cmd or git bash
+2. To clone a repo from github
```
Comment (mkumar04) [inline README.md]: add colon at the end of line

**Review by mkumar04 — APPROVED** [review]
looks good to me.

**Pull request merged**